web-app/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pandas==2.1.4
```

Comment:
Dependencies will be superseded by https://github.com/marcwagn/ultimate_analytics/pull/16